addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18307,3 +18307,35 @@ msgstr ""
 msgctxt "#38041"
 msgid "Mixer"
 msgstr "" 
+
+#. Description of setting "System -> Audio Ouput -> A/V sync method" with label #38200
```

Comment:
that contextual hint is actually wrong, as this is the label of a setting and not it's description :) Also, will users know what "PLL adjustment" means? Could this be phrased less technical as I really doubt the average user knows what PLL is (even I don't know - I only know it's somehow related to playback)